### PR TITLE
Updated Pod Eviction Method for Drain Node Scenarios

### PR DESCRIPTION
**Pull request by shiju-tech**
Updated the method V1beta1Eviction() to V1Eviction() to support v1.22+

### CODE REVIEW

File: chaosk8s/node/actions.py [hunk 1]
```diff
@@ -338,8 +338,8 @@ def drain_nodes(
 
         logger.debug(f"Found {len(eviction_candidates)} pods to evict")
         for pod in eviction_candidates:
-            eviction = client.V1beta1Eviction()
-
+            # eviction = client.V1beta1Eviction()
```
Comment (Lawouach) [inline chaosk8s/node/actions.py:341]: You may remove that line rather than commenting it :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### Updated
+
+-Updated Pod Eviction Method for Drain Node Scenarios
+
```
Comment (Lawouach) [inline CHANGELOG.md:6]: Good catch to add to the changelog. It's often overlooked.

Please simply state you updated to API/v1 as per the lifecycle of Kubernetes API which states the pod eviction has been V1beta1Eviction was deprecated and removed in Kubernetes 1.22.

**Review by Lawouach — COMMENTED** [review]
Good start.

Please make sure to sign your commits so the DCO is happy. Simply add a `-s` to your `git commit` command.

File: chaosk8s/node/actions.py (hunk 1 above)
Comment (shiju-tech) [reply chaosk8s/node/actions.py]: Sure

**Pull request closed without merging**